website/static/data/plugin.json

```diff
@@ -0,0 +1,237 @@
+[
```

Comment:
Please format this file by using `2 spaces`

Reply:
Maybe we could generate this file automatically in the future

Reply:
done